modules/apps/journal/journal-web/src/main/resources/META-INF/resources/edit_folder.jsp

```diff
@@ -157,6 +159,7 @@ renderResponse.setTitle(title);
 				%>
 
 				<liferay-frontend:resource-selector
+
```

Comment:
@gergelyszalay, this was fixed in another commit, but maybe we can rebase and remove this change